src/Documentation/Advanced-Concepts/Heterogeneous-Silos.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: Heterogeneous silos
+---
+
+# Heterogeneous silos
+On a given cluster, silos can support a different set of grain types:
```

Comment:
It doesn't appear in the raw view, but when you click on "Change the file using the online editor" you can see it. Bizarre.